### PR TITLE
Lint with tslint

### PR DESCRIPTION
**Pull request by mlonn**

### CODE REVIEW

**Comment by chraker** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AABB.ts](https://github.com/SoftwareEvolutionProject/2017Group1/blob/98befc1057548571f288ec9a06999cdba8c5c915/frontend/web/src/app/stl-viewer/classes/AABB.ts): file should end with a newline [rule](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS113)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by mlonn** [comment]
More linting to do

File: frontend/web/src/app/components/sidemenu/sidemenu.component.ts [hunk 1]
```diff
@@ -20,22 +20,22 @@ export class SidemenuComponent implements OnInit, AfterViewInit {
...
       _self.hamburger_cross();
     });
 
-    $('[data-toggle="offcanvas"]').click(function() {
+    $('[data-toggle="offcanvas"]').click(() =>  {
       $('#wrapper').toggleClass('toggled');
     });
 
-    $('[data-toggle="collapse"]').click(function(){
+    $('[data-toggle="collapse"]').click(function() {
```
Comment (JBerlin84) [inline frontend/web/src/app/components/sidemenu/sidemenu.component.ts:31]: Varför inte ersätta function() med () => som på alla andra ställen?

Comment (mlonn) [reply frontend/web/src/app/components/sidemenu/sidemenu.component.ts:31]: funkade inte där, i guess pga jQuery eller nått

File: frontend/web/src/app/stl-viewer/classes/Viewer.ts [hunk 2]
```diff
@@ -136,31 +136,29 @@ export class Viewer {
...
-                RenderMode:			parameters.RenderMode || 'flat',
-                Definition:			parameters.Definition || 'standard',
-                FaceCulling:		parameters.FaceCulling || 'on',
-                MipMapping:			parameters.MipMapping || 'off',
-                CreaseAngle:		parameters.CreaseAngle || -180,
-                SphereMapUrl:		parameters.SphereMapUrl || '',
-                ProgressBar:		parameters.ProgressBar || 'on',
-                Renderer:			parameters.Renderer || '',
-                LocalBuffers:		parameters.LocalBuffers || 'retain'
+                SceneUrl:            parameters.SceneUrl || '',
```
Comment (JBerlin84) [inline frontend/web/src/app/stl-viewer/classes/Viewer.ts:142]: Bode rada upp snyggt här när du ändå tagit dig att byta ut tab mot space. ;)

**Review by JBerlin84 — APPROVED** [review]

**Pull request merged**